packages/spaces/commands/vpn/info.js

```diff
@@ -25,45 +25,46 @@ function displayVPNInfo (space, info) {
     return colored
   }
 
-  cli.styledHeader(`${space} VPN Info`)
+  cli.styledHeader(`${name} VPN Info`)
   cli.styledObject({
     ID: info.id,
     'Public IP': info.public_ip,
     'Routable CIDRs': format.CIDR(info.routable_cidrs),
-    State: `${sF(info.state)}`,
-    'Provisioning Status': info.status,
+    'Status': info.status,
     'Status Message': info.status_message
-  }, ['ID', 'Public IP', 'Routable CIDRs', 'State', 'Provisioning Status', 'Status Message'])
+  }, ['ID', 'Public IP', 'Routable CIDRs', 'State', 'Status', 'Status Message'])
 
   // make up tunnel IDs
   info.tunnels.forEach((val, i) => { val.tunnel_id = 'Tunnel ' + (i + 1) })
-  cli.styledHeader(`${space} Tunnel Info`)
+  cli.styledHeader(`${name} Tunnel Info`)
   cli.table(info.tunnels, {
     columns: [
       {key: 'tunnel_id', label: 'VPN Tunnel'},
-      {key: 'outside_ip_address', label: 'IP Address'},
+      {key: 'ip', label: 'IP Address'},
       {key: 'status', label: 'Status', format: status => sF(status)},
       {key: 'last_status_change', label: 'Status Last Changed'},
       {key: 'status_message', label: 'Details'}
     ]
   })
 }
 
-function render (space, info, flags) {
+function render (space, name, info, flags) {
   if (flags.json) {
     cli.styledJSON(info)
   } else {
-    displayVPNInfo(space, info)
+    displayVPNInfo(space, name, info)
   }
 }
 
 function * run (context, heroku) {
   let space = context.flags.space || context.args.space
+  let name = context.flags.name || context.args.name
   if (!space) throw new Error('Space name required.\nUSAGE: heroku spaces:vpn:info --space my-space')
+  if (!name) throw new Error('VPN connection name required.\nUSAGE: heroku spaces:vpn:info --space my-space --name vpn-connection-name')
```

Comment:
I highly recommend lifting @bernerdschaefer's helper function for this:

```node
function check (val, message) {
  if (!val) throw new Error(`${message}.\nUSAGE: heroku spaces:vpn:destroy --space example-space vpn-connection-name`)
}
```

It helps clean these blocks up nicely!